core/shared/src/main/scala/scalaz/zio/ZSchedule.scala

```diff
@@ -654,33 +665,35 @@ trait Schedule_Functions extends Serializable {
    * </pre>
    */
   final def spaced(interval: Duration): Schedule[Any, Int] =
-    forever.delayed(_ + interval)
+    forever.delayed(d => d + relative(interval))
 
   /**
    * A schedule that always recurs, increasing delays by summing the
    * preceding two delays (similar to the fibonacci sequence). Returns the
    * current duration between recurrences.
    */
-  final def fibonacci(one: Duration): Schedule[Any, Duration] =
-    delayed(unfold[(Duration, Duration)]((Duration.Zero, one)) {
-      case (a1, a2) => (a2, a1 + a2)
-    }.map(_._1))
+  final def fibonacci(one: Duration): Schedule[Any, Delay] =
```

Comment:
Try to return `Duration` here instead of `Delay`, by running the `Delay`. We might need to add `mapM` to schedule to do that easily.